unicodetools/src/main/resources/org/unicode/text/UCD/UnicodeInvariantTest.txt

```diff
@@ -530,7 +530,10 @@ Show [\u20b9]
 # exceptions. Should such exceptions arise, they can be added to the definition of
 # $nonAlphabeticBindus to avoid a failure on this test.
 Let $nonAlphabeticBindus = []
-[\p{InSc=Bindu} - $nonAlphabeticBindus - \p{Alphabetic}] = [] 
+[\p{InSc=Bindu} - \p{Alphabetic}] = $nonAlphabeticBindus
+
+Let $nonAlphabeticDependentVowels = [\N{ORIYA SIGN OVERLINE}\N{THAI CHARACTER MAITAIKHU}\N{LIMBU SIGN KEMPHRENG}\N{SHARADA VOWEL MODIFIER MARK}\N{SHARADA EXTRA SHORT VOWEL MARK}]
```

Comment:
Where does this set come from? Did you just try to assert that dependent vowels should be Alphabetic, and you found that these ones are not?
Please say something about this set in the PR description.

FYI @Ken-Whistler

Reply:
> Where does this set come from? Did you just try to assert that dependent vowels should be Alphabetic, and you found that these ones are not?

Yes.

Reply:
Added a footnote in the PR description.

I hesitated to add an invariant that they are all diacritics. Maybe I should.